share/spice/transit/septa/transit_septa.css

```diff
@@ -59,5 +59,5 @@
 .tile--septa .septa__cancelled-tile .septa__time,
 .tile--septa .septa__cancelled-tile .septa__details,
 .tile--septa .septa__cancelled-tile .septa__status {
-    color: #bbb;
-}
\ No newline at end of file
+    color: #5B9E4D;
```

Comment:
Hey @javathunderman, this is actually the color of the text when the train is cancelled. We gray out the tile when that happens, and so the color of the text is changed. It's probably left better as gray instead of green :smile: